libclamav_rust/src/cdiff.rs

```diff
@@ -960,7 +1027,10 @@ fn process_line(ctx: &mut Context, line: &[u8]) -> Result<(), InputError> {
             cmd_move(ctx, move_op)
         }
         b"CLOSE" => cmd_close(ctx),
-        b"UNLINK" => cmd_unlink(ctx),
+        b"UNLINK" => {
+            let unlink_op = UnlinkOp::new(remainder.unwrap())?;
+            cmd_unlink(ctx, unlink_op)
+        }
         _ => Err(InputError::UnknownCommand(
             String::from_utf8(cmd.to_owned()).unwrap(),
```

Comment:
Not your code, but there's the potential to thrown an error while building the error here. I would recommend using `::from_utf8_lossy()` to avoid panicking if the content is non-UTF-8

Reply:
Good idea.  I've made the change.  Merging.